src/main/java/world/bentobox/bentobox/commands/BentoBoxCommand.java

```diff
@@ -16,8 +16,10 @@ public BentoBoxCommand() {
 
     @Override
     public void setup() {
+        setPermission("bentobox.admin");
```

Comment:
Only BentoBoxReloadCommand should need the permission. "version" and "about" are doing no harm.

Reply:
ok